src/devana/syntax_abstraction/organizers/codecontainer.py

```diff
@@ -34,6 +35,16 @@ def from_cursor(cls, cursor: cindex.Cursor, parent: Optional = None) -> Optional
     def create_default(cls, parent: Optional = None) -> Any:
         return cls(None, parent)
 
+    @classmethod
+    @init_params(skip={"cls", "parent"})
+    def from_params( # pylint: disable=unused-argument
+            cls,
+            parent: Optional = None,
+            content: Optional = None,
+            namespace: Optional = None
+    ) -> "CodeContainer":
+        return cls(None, parent)
```

Comment:
I'm not a big fan of this notation. Ultimately it's something I can accept but if you feel like playing/testing you could think of a property on the class that would generate a method that would take all these parameters - then in this function we would call it explicitly.
But that's just thinking out loud.

Reply:
I'll play with it, but probably will leave it as it is